plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraAnimationsPluginImpl.kt

```diff
@@ -805,7 +807,9 @@ class CameraAnimationsPluginImpl : CameraAnimationsPlugin {
     for (cameraAnimator in animators) {
       if (cameraAnimator is CameraAnimator<*>) {
         cameraAnimator.isInternal = true
```

Comment:
Should flag be always isInternal == true then?

Reply:
yes, I assume so, this flag is only checked internally to handle the animation end. The bug is only associated with the owner of the animator.